install.sh

```diff
@@ -400,12 +399,27 @@ get_architecture() {
 
 resolve_version() {
   local _requested_version=$1
-  local _requested_ref=$2
+  
+  local _ref
+  local _repo
+
+  if grep -q "nightly"<<<"$_requested_version"; then 
+    _repo="$SCARB_NIGHTLIES_REPO"
+    _ref="tag/${_requested_version}"
+  else
+    _repo="$SCARB_REPO"
+    if grep -q "latest"<<<"$_requested_version"; then
+      _ref="latest"
+    else
+      _ref="tag/v${_requested_version}"
+    fi
+  fi
 
   local _response
 
-  say "retrieving $_requested_version version from ${SCARB_REPO}..."
-  _response=$(ensure curl -Ls -H 'Accept: application/json' "${SCARB_REPO}/releases/${_requested_ref}")
+  say "${_repo}/releases/${_ref}"
```

Comment:
Just making sure, is this intentional?

Reply:
Yeah, I don't think so

Reply:
It's good to have a second pair of eyes 😅, totally unintentional, thanks for the catch!